tests/common/plugins/sanity_check/checks.py

```diff
@@ -206,6 +206,21 @@ def _check_bgp_status_helper():
                 check_result['failed'] = False
             return not check_result['failed']
 
+        def _check_bgp_router_id(tbinfo):
+            duthost = kwargs['node']
```

Comment:
Looks like there is duplication of whole logic here. So can we put this into commons and import at both the places?